src/Sylius/Bundle/AdminApiBundle/Command/CreateClientCommand.php

```diff
@@ -15,13 +15,25 @@
 
 use FOS\OAuthServerBundle\Model\ClientManagerInterface;
 use Sylius\Bundle\AdminApiBundle\Model\Client;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-final class CreateClientCommand extends ContainerAwareCommand
+final class CreateClientCommand extends Command
 {
+    /**
+     * @var ClientManagerInterface
+     */
+    private $clientManager;
+
+    public function __construct(string $name = null, ClientManagerInterface $clientManager)
```

Comment:
Unfortunately, adding a mandatory argument to the constructor method is considered a BC break.

Reply:
I can add a separate class and override service definition in the configuration if current Symfony version is 4.1. 

This class will be deprecated then.

Reply:
What about this solution?